easybuild/easyconfigs/q/QuantumESPRESSO/QuantumESPRESSO-6.0-foss-2016b.eb

```diff
@@ -0,0 +1,62 @@
+name = 'QuantumESPRESSO'
+version = '6.0'
+
+homepage = 'http://www.pwscf.org/'
+description = """Quantum ESPRESSO  is an integrated suite of computer codes
+ for electronic-structure calculations and materials modeling at the nanoscale.
+ It is based on density-functional theory, plane waves, and pseudopotentials
+  (both norm-conserving and ultrasoft)."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'usempi': True, 'openmp': True}
```

Comment:
@ocaisa hmm, `openmp` is not enabled in the QE 6.0 easyconfig using intel/2016b that was added in #3647, but it probably should be, can you enable it there too please?